Tests/test_PDB_StructureAlignment.py

```diff
@@ -48,11 +48,68 @@ def test_StructAlign(self):
             chain2_A = m2["A"]
             self.assertEqual(chain1_A[202].get_resname(), "ILE")
             self.assertEqual(chain2_A[202].get_resname(), "LEU")
-            self.assertEqual(chain1_A[291].get_resname(), chain2_A[180].get_resname())
             self.assertNotEqual(
                 chain1_A[291].get_resname(), chain2_A[181].get_resname()
             )
 
+    def test_StructAlign_msa_vs_alignment_objects(self):
```

Comment:
I'd divide this in multiple tests:

1. Testing that MultipleSeqAlignment and Alignment work the same
2. Testing that providing a custom alignment (running PairwiseAligner) and passing it, and using the new mode of doing automatic alignment work and produce identical results.